### ISSUE TITLE
Feature Request: Bypass adsites like adf.ly

### ISSUE THREAD

**Issue by WolfspiritM** [gorhill/uMatrix#86]
Since uMatrix blocks sites like adf.ly it would be nice to have a way to define a way to "bypass" these sites. As adfly for example has URLs like:
http://adf.ly/123123123/example.com
It could be easy for uMatrix to just remove the URL and the first part of the Path to get the expected URL and just redirect to example.com instead of displaying "adf.ly blocked by μMatrix".
Maybe for some "advanced settings" which allows defining these things with regular expressions.

**Comment by gorhill** [comment]
Parsing/redirecting URLs is not something that fits uMatrix's purpose.

**Comment by softwarestake** [comment]
thanks for sharing but m finding adfly bypass api find all over internet but did not found -_- 
http://www.apkgamestic.com

**Comment by d-kr** [comment]
@softwarestake : Just google for unshortener or untinyer. I found at least four chrome extension who selfclaim that they can bypass many url shortener. (But I did not test any on those)

**Comment by gorhill** [comment]
@d-kr Not sure but I think @softwarestake was just spamming. I removed the link he put in which was completely irrelevant.
